src/librbd/image/RemoveRequest.cc

```diff
@@ -621,6 +627,24 @@ void RemoveRequest<I>::handle_remove_v1_image(int r) {
   if (!m_old_format) {
     remove_v2_image();
   }
+
+}
+
+template<typename I>
+void RemoveRequest<I>::remove_image_cache() {
```

Comment:
Yes, but it's missing the point that in a real-world scenario, the "rbd rm" operation will most likely never be executed where the image was being used. Therefore, this logic will not serve any purpose and will not clean up the cache.